api/data.go

```diff
@@ -116,3 +139,31 @@ func (api *API) GetCacheTime(ctx context.Context, w http.ResponseWriter, req *ht
 		http.Error(w, "Internal Server Error", http.StatusInternalServerError)
 	}
 }
+
+func isValidCacheTime(cacheTime *models.CacheTime) error {
+	errs := []error{}
```

Comment:
I think this can also be rewritten as `var errs []error`, but my Go is not good enough to know which one is preferred (or even if they're truly equivalent), but I figured I should mention it just in case you know the difference.

Reply:
I believe the difference is that `var errs []error` will be initialised as a nil value, where as `errs := []error{}` will be initialised as an empty slice `[]`. However as we are just checking the length here to confirm if there were no errors, they can be used interchangeably.